### PR TITLE
New version cleaner than the latest

### PR DESCRIPTION
**Pull request by CodingDotExe**
# What's new ?
- The bot is now working using functions
- Bugs are fixed
- Verifying for every task the permissions of the bot, send a message in console if the bot can't make something

### I was proud to help

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
-AND THIS CODE WILL BAN ALL MEMBERS !
-Packages : Discord.js to install npm i discord.js
-ENJOY
-EVIL
-BY : ME !
+# Discord Bots Hack
+
+
+##### How to use ?
+1.  Download and configure [nodsjs](https://nodejs.org/en/) if not yet
```
Comment (Androz2091) [inline README.md]: Small spelling error (nodsjs)

Comment (CodingDotExe) [reply README.md]: Oh yeah, didn't see, thanks :slightly_smiling_face:

**Comment by Bowlingtoolkit** [comment]
Thank you for developing my code, I liked your changes .. 👍  :)

**Comment by CodingDotExe** [comment]
> Thank you for developing my code, I liked your changes .. 👍 :)

Thank you for accept them and to add me in credit 💫 :) call me for next issues if you need help 😉

**Pull request merged**